### ISSUE TITLE
High disk I/O usage

### ISSUE THREAD

**Issue by Daniel15** [theotherp/nzbhydra2#917]
NZBHydra is consistently reading/writing at ~60MB/s even when it's just sitting idle.

![image](https://github.com/theotherp/nzbhydra2/assets/91933/872067db-08d7-492e-84d5-4b5e06386f5a)

System load dropped quite a bit once I stopped my NZBHydra Docker container.

- Docker container: `lscr.io/linuxserver/nzbhydra2`, latest version as of today
- OS: Unraid 6.12.4
- debuginfos: [nzbhydra-debuginfos-2024-02-18-18-20.zip](https://github.com/theotherp/nzbhydra2/files/14325598/nzbhydra-debuginfos-2024-02-18-18-20.zip)

**Comment by theotherp** [comment]
Yeah, that's insane. I know of some cases where the DB causes high IO but I don't think I've ever seen it this high.

Thanks for the debug infos but I need some more data. Enable the display of advanced options in the config and then enable the logging marker for performance:
![image](https://github.com/theotherp/nzbhydra2/assets/14368889/2e0b31b4-e422-4cf6-9420-8579f5628c39)

Then when the IO usage is high go to the debug infos section and make a screenshot. It should look a bit like this:
![image](https://github.com/theotherp/nzbhydra2/assets/14368889/3f9c8858-8e10-4efa-b397-93efba0231c3),
only "hopefully" with more activity. That may allow me to see if it's the database thread doing the work.

Then create a thread dump by running this curl: ` curl 'http://127.0.0.1:5076/actuator/threaddump' -i -X GET -H 'Accept: text/plain' > threaddump.txt` (using whatever host and IP is correct) and post the resulting txt file.

**Comment by Daniel15** [comment]
I had to reinstall all my Docker containers and now it's not doing it any more. I see zero I/O from NZBHydra. I'll keep an eye out for it and follow your steps if I see it again. Maybe it only happens after running for a while.
